src/main/webapp/app/shared/modal/AddMutationModal.tsx

```diff
@@ -151,24 +157,24 @@ function AddMutationModal({
         excluding: alteration.excluding?.map(ex => convertAlterationToAlterationData(ex)) || [],
         genes: alteration?.genes || [],
         proteinChange: alteration?.proteinChange,
-        proteinStart: alteration?.proteinStart === -1 ? null : alteration?.proteinStart,
-        proteinEnd: alteration?.proteinEnd === -1 ? null : alteration?.proteinEnd,
+        proteinStart: alteration?.proteinStart === -1 ? undefined : alteration?.proteinStart,
+        proteinEnd: alteration?.proteinEnd === -1 ? undefined : alteration?.proteinEnd,
         refResidues: alteration?.refResidues,
         varResidues: alteration?.varResidues,
       };
     }
 
     async function setExistingAlterations() {
-      if (mutationToEdit.alterations?.length > 0) {
-        setTabStates(mutationToEdit.alterations.map(alt => convertAlterationToAlterationData(alt)));
+      if ((mutationToEdit?.alterations?.length ?? 0) > 0) {
```

Comment:
Nit: Not necessary since `null > 0 === false`

Reply:
That's not a nit. It's a fair comment.
The strict checking counts greater than/less than logic as an error if a `null`/`undefined` is involved.